sdk/core/test/unit/services/WalletStorageService.test.ts

```diff
@@ -149,6 +171,22 @@ describe('WalletStorageService', () => {
     })
   })
 
+  describe('#getAllCredentialsWithBadEncryption', () => {
```

Comment:
I'd like to keep `describe` block with method name only, and move this detail (BadEncryption) to the `it` block

So please move you it block under `#getAllCredentials`

Reply:
ok